web_src/js/features/imagediff.js

```diff
@@ -255,13 +255,10 @@ export function initImageDiff() {
         width: sizes.size2.width * factor + 2,
         height: sizes.size2.height * factor + 2
       });
+
       sizes.image2.parent().parent().css({
         width: sizes.max.width * factor + 2,
-        height: sizes.max.height * factor + 2
-      });
-      $container.find('.onion-skin').css({
-        width: sizes.max.width * factor + 2,
-        height: sizes.max.height * factor + 4
+        height: sizes.max.height * factor + 2 + 20,
```

Comment:
Where is this `+20` coming from?
This looks like a magic constant to me.
I mean, I can at least guess that the `+2` everywhere else is so that a `1px` border can be drawn, but this `+20`?

Reply:
To fix "including regressions from https://github.com/go-gitea/gitea/pull/22959"

The inner element is `position: absolute`, so the container's height should be large enough. I will add a comment.

Reply:
Comments are added.